chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertificationBodyManagerImpl.java

```diff
@@ -122,6 +122,9 @@ public CertificationBodyDTO update(final CertificationBodyDTO acb)
     @Transactional
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).CERTIFICATION_BODY, "
             + "T(gov.healthit.chpl.permissions.domains.CertificationBodyDomainPermissions).RETIRE)")
+    @CacheEvict(value = {
+            CacheNames.COLLECTIONS_LISTINGS
```

Comment:
Need one of these on ACB update as well, not just retire/unretire

Reply:
I'm not familiar with what you mean. Is there a snippet of code for ACB update I can be pointed to?